checkPreRequisites.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+echo '-------------------------------------------'
+nodeVersion=$(node --version)
+
+if [[ nodeVersion > 4.0 ]] || [[ nodeVersion == 4.0 ]]; then
```

Comment:
Please update the minimum version to **8.0**

Reply:
done